cookbooks/swift/templates/default/etc/swift/account-server/server.conf-template.erb

```diff
@@ -3,7 +3,7 @@ disable_fallocate = true
 workers = 1
 
 [pipeline:main]
-pipeline = recon account-server
+pipeline = backend_ratelimit recon account-server
```

Comment:
the upstream sample conf (and prod) has backend_ratelimit immediately before the server app (for no particular reason, but let's be consistent)